tests/test_runner.sh

```diff
@@ -32,11 +32,15 @@ for f in $(
     echo "... Ok" >&2
 done
 
+set -x
 echo "Installing Moby packages..." >&2
-sshCmd '/opt/moby/install.sh' || exit
+sshCmd "eval \"${TEST_EVAL_VARS}\"; /opt/moby/install.sh" || exit
```

Comment:
If the vars were space separated & without the export, this could be
```
sshCmd "env -S \"${TEST_EVAL_VARS}\" /opt/moby/install.sh"
```

Which avoids the `eval` and simplifies setting up the vars. Not sure if it's any better but figured I'd mention it.

Reply:
TIL. I like this better than `eval` for sure.
Fixed.